application/web/js/utilities.js

```diff
@@ -36,6 +36,22 @@
 
             return attrValue;
         },
+        /**
+         * Generate an error if the passed value is not an integer
+         * @param val Note any number of arguments can be passed, all will be validated
+         */
+        assertInteger : function (val) {
```

Comment:
We should move to compiling our JS so this kind of thing can just be a regular variadic function